tftbase.py

```diff
@@ -71,6 +71,30 @@ def get_tft_image_pull_policy() -> str:
 T = typing.TypeVar("T")
 
 
+def eval_binary_opt_in(
```

Comment:
This name is a bit confusing. I am not sure what to call this but does it follow AND/NAND/NOR/OR logic?